modules/ROOT/pages/getting.started.dbt-feast-teradata-pipeline.adoc

```diff
@@ -0,0 +1,310 @@
+= A Data pipeline with DBT+FEAST on Teradata
+
+:experimental:
+:page-author: Ravi Chr
```

Comment:
Is that your real name :)?

Reply:
My full name is Ravi Chillanki 
Pen name is Ravi Chr :)